etc/profile-a-l/gzdoom-common.profile

```diff
@@ -0,0 +1,60 @@
+# Firejail profile for gzdoom-common
+# This file is overwritten after every install/update
+# Persistent local customizations
+include gzdoom-common.local
+# Persistent global definitions
+# added by caller profile
+#include globals.local
+
+noblacklist ${HOME}/.local/share/games/doom
+
+# Allow /bin/sh (blacklisted by disable-shell.inc)
+include allow-bin-sh.inc
+
+include disable-common.inc
+include disable-devel.inc
+include disable-exec.inc
+include disable-interpreters.inc
+include disable-proc.inc
+include disable-programs.inc
+include disable-shell.inc
+include disable-xdg.inc
+
+mkdir ${HOME}/.local/share/games/doom
+whitelist ${HOME}/.local/share/games/doom
+whitelist /usr/local/share/doom
+whitelist /usr/local/share/games/doom
```

Comment:
```suggestion
```

There are no other profiles using /usr/local/share, so unless there is a
specific reason for it, it would be good for consistency to remove it.